cli/lib/kontena/cli/apps/yaml/validator_v2.rb

```diff
@@ -0,0 +1,70 @@
+require 'dry-validation'
+require_relative 'validator'
+
+module Kontena::Cli::Apps
+  module YAML
+    class ValidatorV2
+      require_relative 'validations'
+      include Validations
+
+      VALID_KEYS = %w(
+      affinity build cap_add cap_drop command deploy depends_on env_file environment extends external_links
+      image links logging network_mode pid ports volumes volumes_from cpu_shares
+      mem_limit memswap_limit privileged stateful user instances hooks secrets
+      ).freeze
+
+      UNSUPPORTED_KEYS = %w(
+      cgroup_parent container_name devices dns dns_search tmpfs entrypoint
+      expose extra_hosts labels log_driver log_opt net networks security_opt stop_signal ulimits volume_driver
+      cpu_quota cpuset domainname hostname ipc mac_address
+      read_only restart shm_size stdin_open tty working_dir
+      ).freeze
+
```

Comment:
`log_opts` and `log_driver` are not supported on version 2: https://docs.docker.com/compose/compose-file/#log-driver

Are there any other that I might have missed?